src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container container-big">
```

Comment:
Your html have a lot of redundant classes which not use in css

Reply:
Do you mean not to use the class name "container"? Or do not write classes that are not in css (For example class=“form form-small” I didn’t use it, but I wrote in the index.html)?